src/sass/_config.scss

```diff
@@ -62,6 +62,8 @@ $fontSizes: (
   small: 0.8125rem,   // 13px
   medium: 0.9375rem,  // 15px
   large: 3.625rem,    // 58px
+  xlarge: 4.5rem,     // 72px
+  xxlarge: 5rem,      // 80px
```

Comment:
Needed for text-bit.

No idea how to make all font size names adhere to `2D` convention